File: stateLevel/java/slope.java
import java.util.Scanner;
import java.lang.Math;

public class slope {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input1 = scanner.nextLine().split(",");
        String[] input2 = scanner.nextLine().split(",");

        try {
            double x1 = Double.parseDouble(input1[0]);
            double y1 = Double.parseDouble(input1[1]);
            double x2 = Double.parseDouble(input2[0]);
            double y2 = Double.parseDouble(input2[1]);

            if (x1 == x2 && y1 == y2) {
                System.out.println("Unable to calculate slope.");
            } else {
                double slope = (y2 - y1) / (x2 - x1);
                System.out.printf("%.1f%n", slope);
            }
        } catch (NumberFormatException e) {
            System.out.println("Unable to calculate slope.");
        }
    }
}

File: stateLevel/py/slope.py
def calculate_slope(point1, point2):
    try:
        x1, y1 = point1
        x2, y2 = point2
        if x1 == x2:
            raise ValueError("Unable to calculate slope.")
        slope = (y2 - y1) / (x2 - x1)
        return round(slope, 1)
    except ValueError as e:
        return str(e)

def get_input():
    point1 = input().split(',')
    point2 = input().split(',')
    return [float(point1[0]), float(point1[1])], [float(point2[0]), float(point2[1])]

def main():
    point1, point2 = get_input()
    slope = calculate_slope(point1, point2)
    print(slope)

if __name__ == "__main__":
    main()